packages/storage/src/indexed-db/indexed-db.test.ts

```diff
@@ -53,16 +53,19 @@ import {
   DutchAuctionDescription,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/auction/v1alpha1/auction_pb';
 import { StateCommitment } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/crypto/tct/v1/tct_pb';
+import { ChainRegistryClient } from '@penumbra-labs/registry';
 
 describe('IndexedDb', () => {
   // uses different wallet ids so no collisions take place
+  const registryClient = new ChainRegistryClient();
+  const chainId = 'penumbra-testnet-deimos-6';
```

Comment:
Optional
You could use mock for `ChainRegistryClient` instead, then the indexed-db tests would not depend on registry. And the tests won't break if the registry no longer contains data for the test `chain-id`.

Reply:
I started down this track, but realized that we'd need to implement an interface, but the registry wasn't currently exposing one. Given time constraints, just used a chain id we know is there. Can revisit this later though.